Makefile

```diff
@@ -91,5 +93,10 @@ endif
 
 .PHONY: release
 release:
-	tar –cf dashboard.tar ./output/*
+	tar -zcvf $(RELEASE).tgz ./output/*
```

Comment:
NO.
`release` is ONLY for release manager, who ONLY release source code packge.

Reply:
Ok, It seems that `make release` should not be used here.
The `output` directory is not source code. They are the result of compilation, including manager and web.
Maybe `make publish` is more suitable. Ref to some web projects , Such as elementUI.